### ISSUE TITLE
onClick showing the wrong word

### ISSUE THREAD

**Issue by Zukeh** [timdream/wordcloud2.js#66]
Hi,

Thank you for the great library!

I'm having a small problem, my onClick event is showing the wrong word.
You can see the problem here: http://jsbin.com/fojiwituxa/edit?js,output

My initialization code is very simple:

```
WordCloud(document.getElementById('c'), {
                list: list_english,
                ellipticity: 1.5,
                hover: function(item, dimension, event) {
                    //console.log(event);
                },
              click: function(item) {
                alert(item[0] + ': ' + item[1]);
              }
            });
```

**Comment by timdream** [comment]
I can confirm this is a problem. It looks like the internal coordinate to figure out the hovered item didn't return the right number. Specifically, the function here:

https://github.com/timdream/wordcloud2.js/blob/d9dc8c5cab7d8e9ce8073ef2a08b7ac781f5e487/src/wordcloud2.js#L371-L384

Spend some time inspecting this function with breakpoints or console.log() should find the cause here.

**Comment by timdream** [comment]
I can confirm the issue is fixed (per your JSBin example) on the latest version.

There was a regression caused by 625d6f9 and fixed by 0638574. Please update to v1.0.3. Thanks!
